config/vx-precinct-scanner.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=VotingWorks Precinct Scanner
+
+[Service]
+Type=simple
+User=vx-services
+Environment=VX_CONFIG_ROOT=/vx/config
```

Comment:
should this also have an environment variable for `VX_MACHINE_TYPE` ?

Reply:
and then for the `bsd` service, the other value?

Reply:
This happens automatically because of this line in `run-*.sh`:

```sh
source ${CONFIG}/read-vx-machine-config.sh
```

Reply:
oh duh